docs/ui/schema_viewer.rst

```diff
@@ -0,0 +1,69 @@
+.. _ref_ui_schema_viewer:
+
+=============
+Schema Viewer
+=============
+
+.. image:: images/schema_viewer.png
+    :alt: The Schema Viewer page in the EdgeDB UI. The icon is two small
+          squares of blue and orange, connected by a purple line. A small
+          user-defined sample schema is shown with two concrete types
+          called Book and Library, along with an abstract type called
+          HasAddress that is extended by the Library type.
+    :width: 100%
+
+The UI schema viewer allows you to see your schema either as text, graph,
+or both.
+
+Text view
+---------
+
+The UI's text view is similar to what is produced in the CLI through the
+``describe schema as sdl`` command, but includes inherited properties and
+links.
+
+The schema viewer shows the user's defined schema by default under the
+name ``User``, and also includes ``Stdlib`` and ``System`` as a convenience.
```

Comment:
Let's add a screenshot of this dropdown menu in the larger context of the UI but maybe with everything around it dimmed a bit